rpcs3/Emu/RSX/RSXThread.cpp

```diff
@@ -2029,6 +2033,7 @@ namespace rsx
 				// These are single byte formats, but inverted order (BGRA vs ARGB) when passed via registers
 				to_swap_bytes = (layout.attribute_placement[index] == attribute_buffer_placement::transient);
 				break;
+			default: break;
```

Comment:
Newline. If the preceding blocks have break on newline, just make default also break on newline for consistency. Also really helps with debugging as there is a line number to place a breakpoint on.